README.md

```diff
@@ -1,13 +1,19 @@
 # Origin-Aggregated-Logging [![Build Status](https://ci.openshift.redhat.com/jenkins/buildStatus/icon?job=test-origin-aggregated-logging)](https://ci.openshift.redhat.com/jenkins/job/test-origin-aggregated-logging)
 
-This repo contains the image definitions of the components of the logging
-stack as well as tools for building and deploying them.
+This repo contains the image definitions for the components of the logging
+stack as well as tools for building and deploying them.  The logging subsystem 
+consists of multiple [components](#Components) commonly abbreviated as the "ELK" 
```

Comment:
I think we should change the order of this; its slightly confusing if you aren't familiar with the ELK|EFK stack and may think this is an "ELK" stack...
Maybe something like "consists of multiple components abbreviated as the "EFK" stack (Elasticsearch, Fluentd, Kibana)."